stack.yaml

```diff
@@ -44,9 +44,9 @@ extra-deps:
     - persistent-sqlite
     - persistent-template
 
-# cardano-addresses-2.1.0
+# cardano-addresses
 - git: https://github.com/input-output-hk/cardano-addresses
-  commit: e0ab4587266430a08734e1aec1c29d261a9a3b70
+  commit: 02fb4ec7db7c55334ab2216c94cf70b209a33885
```

Comment:
We need to use released version of cardano-addresses. There are some cleanup to be done in cardano-addresses before we can release a 3.0.0 version. Let's discuss that on Slack.